### PR TITLE
fix:  removed price components. Altered QA.

### PR DESCRIPTION
**Pull request by PalmN72**
As per the latest issue #651 . We have altered the website to reflect the new price model, ie free. Removed the price components, changed the QA.

### CODE REVIEW

File: apps/website/components/FunFacts.tsx [hunk 1]
```diff
@@ -12,10 +11,6 @@ const FUNFACTS_DATA = [
     count: 7,
     title: 'år att utveckla',
   },
-  {
-    count: price,
```
Comment (kajetan-kazimierczak) [inline apps/website/components/FunFacts.tsx:16]: I think this should be left, but price changed do 0.

File: apps/website/components/NavLinks.tsx [hunk 2]
```diff
@@ -66,11 +66,6 @@ const NavLinks = ({ onClick }: NavLinksProps) => {
           {intl.formatMessage({ id: 'navigation.screenshots' })}
         </Link>
       </li>
-      <li>
-        <Link to="vad-kostar-det" href="/#vad-kostar-det">
-          {intl.formatMessage({ id: 'navigation.whatdoesitcost' })}
```
Comment (kajetan-kazimierczak) [inline apps/website/components/NavLinks.tsx:71]: This should also be left in.

File: apps/website/components/QA.tsx [hunk 3]
```diff
@@ -251,13 +251,11 @@ const QA = () => {
...
         </h3>
         <p>
-          Appen kostar {price} kronor. Intäkten registreras i aktiebolaget Not Free
-          Beer som ägs av tre av utvecklarna och går till att täcka kostnader
-          för inköp. Det täcker inte på långa vägar den tid vi lagt ner. Med en
-          låg engångskostnad ökar vi chansen att vi orkar syssla med underhåll
-          och uppdateringar. Vi vill ju ha en stabil lösning som håller. Just nu
-          jobbar vi på egen fritid med något som förbättrar det kommunen lagt en
-          miljard av allmänna medel på.
+          Vi har en Patreon där summan är synlig. Intäkten registreras i
```
Comment (kajetan-kazimierczak) [inline apps/website/components/QA.tsx:254]: Good

File: apps/website/pages/index.tsx [hunk 4]
```diff
@@ -21,7 +20,6 @@ const HomePage: NextPage = () => {
       <CtaTwo />
       <CtaThree />
       <AppShots />
-      <PricingTemp />
```
Comment (kajetan-kazimierczak) [inline apps/website/pages/index.tsx:24]: This should be left in, but the PricingTemp text should be updated

**Comment by kajetan-kazimierczak** [comment]
Thanks. Good PR. I think though that a better way is to show that the price is free, than not showing the pricing information at all (since there was a price before and many people still think there is). I have made some small suggestions.

**Comment by PalmN72** [comment]
Thanks kajetan! We will look over the suggestions and make changes accordingly!

**Comment by kajetan-kazimierczak** [comment]
Great!

**Pull request merged**